source/MRMesh/MRProgressCallback.h

```diff
@@ -9,4 +11,10 @@ namespace MR
 /// \ingroup BasicStructuresGroup
 typedef std::function<bool( float )> ProgressCallback;
 
+// Returns message showed when loading is canceled
+inline std::string getCancelMessage( const std::filesystem::path& path )
+{
+	return "Loading canceled: " + path.string();
```

Comment:
never use `.string()` - it returns not utf8 and throws exceptions, we have `utf8string` for this

Reply:
fixed